### ISSUE TITLE
Why fuzzy-match * doesn't work?

### ISSUE THREAD

**Issue by coolbq** [johnkerl/miller#1341]
os: Windows 10
command: `mlr --csv --mfrom *.csv -- cat`
error: mlr: open *.csv: The filename, directory name, or volume label syntax is incorrect..

why "*.csv" doesn't match multiple csv files on Windows 10 ?

**Comment by johnkerl** [comment]
Hi @coolbq ! Can you show the output of `dir *.csv` in a command window?

**Comment by aborruso** [comment]
I have the same in windows 11. If I run

```
mlr.exe --csv --mfrom *.csv -- cat
```

I have `mlr: open *.csv: The filename, directory name, or volume label syntax is incorrect..`.

![image](https://github.com/johnkerl/miller/assets/30607/489bfdcd-f150-4b13-bc57-0cb3eaa4812e)

**Comment by johnkerl** [comment]
OK I will look -- thank you! -- my Windows machine is not with me at present but I'll check in a few days.

**Comment by coolbq** [comment]
> Hi @coolbq ! Can you show the output of `dir *.csv` in a command window?

![image](https://github.com/johnkerl/miller/assets/2296444/b24148b9-e1c7-4a1d-bc0e-05e335e2026a)

**Comment by johnkerl** [comment]
@coolbq and @aborruso my apologies, I know what this is, and I have known for many, many years -- even before Miller.


The issue -- a standard issue -- is that Unix and Unix-like shells (including Linux and Mac) expand wildcards like `*.csv` _before_ handing off to a program, whereas in Windows, programs have to expand themselves.

I've known this for a very long time -- 20 years easily -- and also, the Windows version of Miller goes back many years as well (although of course not 20). I'm embarrassed to say I didn't think of this earlier -- it should have been a fundamental part of any Windows port for the very start, and not only did I not implement wildcard-expansion for Miller on Windows, I didn't write any test cases, etc.

```
C:\miller\docs\src> mlr --csv cat circle.csv
color,shape,flag,k,index,quantity,rate
red,circle,true,3,16,13.8103,2.9010
yellow,circle,true,8,73,63.9785,4.2370
yellow,circle,true,9,87,63.5058,8.3350
```

```
C:\miller\docs\src> mlr --csv cat circle.csv commas.csv
color,shape,flag,k,index,quantity,rate
red,circle,true,3,16,13.8103,2.9010
yellow,circle,true,8,73,63.9785,4.2370
yellow,circle,true,9,87,63.5058,8.3350

Name,Role
"Xiao, Lin",administrator
"Khavari, Darius",tester
```

```
C:\miller\docs\src> mlr --csv cat *.csv
mlr: open *.csv: The filename, directory name, or volume label syntax is incorrect..
```

**Comment by johnkerl** [comment]
@coolbq @aborruso fix on #1362

**Comment by aborruso** [comment]
> @coolbq @aborruso fix on #1362

It works, thank you very much

**Comment by coolbq** [comment]
> @coolbq @aborruso fix on #1362

Yes, it works now. Thanks alot !

**Comment by coolbq** [comment]
hhm, it seems there is still a further problem for this fuzzy match combination operation. The combination process just stucks there and never end. If stop the command line windows forcefully, you will get a huge result file.

`mlr --csv --mfrom *.txt  -- cat > Comb1.txt`

![image](https://github.com/johnkerl/miller/assets/2296444/ca113a58-72a1-4c0c-9fa2-a4bb81e00dee)
